app/frontend/entrypoints/application.js

```diff
@@ -0,0 +1,72 @@
+// To see this message, add the following to the `<head>` section in your
+// views/layouts/application.html.erb
+//
+//    <%= vite_client_tag %>
+//    <%= vite_javascript_tag 'application' %>
+console.log('Vite ⚡️ Rails')
+
+// If using a TypeScript entrypoint file:
+//     <%= vite_typescript_tag 'application' %>
+//
+// If you want to use .jsx or .tsx, add the extension:
+//     <%= vite_javascript_tag 'application.jsx' %>
+
+console.log('Visit the guide for more information: ', 'https://vite-ruby.netlify.app/guide/rails')
+
+// Example: Load Rails libraries in Vite.
+//
+// import * as Turbo from '@hotwired/turbo'
+// Turbo.start()
+//
+// import ActiveStorage from '@rails/activestorage.vue'
+// ActiveStorage.start()
+//
+// // Import all channels.
+// const channels = import.meta.globEager('./**/*_channel.js')
+
+// Example: Import a stylesheet in app/frontend/index.css
+// import '~/index.css'
+
+// ^^^ Template from Vite. Below is custom code for Limber ^^^
+
+// Import Libraries
+import 'bootstrap'
+import 'popper.js'
+
+// Load all javascript files previously in the app/assets directory
+import '@/javascript/bed_verification.js'
+import '@/javascript/choose_workflow.js'
+import '@/javascript/global_message_system.js'
+import '@/javascript/legacy_scripts_a.js'
+import '@/javascript/merged_plate.js'
+import '@/javascript/multi_plate_pooling.js'
+import '@/javascript/multi_tube_pooling.js'
+import '@/javascript/pooled_tubes_from_whole_plates.js'
+import '@/javascript/session_scripts.js'
+import '@/javascript/state_change_reasons.js'
+import '@/javascript/state_machine.js'
+import '@/javascript/tag_by_tag_plate.js'
+import '@/javascript/tooltips.js'
+// Load all javascript files previously in the app/assets/lib directory, these really should
+// be loaded as required, not globally as previously done.
+import '@/javascript/lib/ajax_support.js'
+import '@/javascript/lib/array_fill_polyfill.js'
+import '@/javascript/lib/disable_enter_key_submit.js'
+
```

Comment:
Based on the comment, should we import key_codes, status_collector, tag_collector, and validator here as well?